bench/wtperf/wtperf.c

```diff
@@ -455,6 +465,21 @@ worker(void *arg)
 		lprintf(cfg, ret, 0, "First transaction begin failed");
 		goto err;
 	}
+	/* Setup truncate values */
+	if (thread->workload->truncate > 0) {
```

Comment:
It's not clear anywhere whether truncate is a count or a boolean. I believe it's a boolean - as such it would be good to make that clear in the configuration API, and to treat it as a boolean in the code (i.e: I'd change this to ```!= 0```).